roles/openshift_node/templates/node.yaml.v1.j2

```diff
@@ -16,8 +16,8 @@ kubeletArguments: {{ openshift.node.kubelet_args | default(None) | to_padded_yam
 masterClientConnectionOverrides:
   acceptContentTypes: application/vnd.kubernetes.protobuf,application/json
   contentType: application/vnd.kubernetes.protobuf
-  burst: 40
-  ops: 20
+  burst: 200
+  qps: 100
```

Comment:
@deads2k  I should've asked before merging, but wanted to verify this value change was intentional.